src/main/scala/daisy/backend/CodeGenerationPhase.scala

```diff
@@ -27,7 +27,7 @@ object CodeGenerationPhase extends DaisyPhase {
   override val definedOptions: Set[CmdLineOption[Any]] = Set(
     StringChoiceOption(
       "lang",
-      Set("C", "Scala", "FPCore"),
+      Set("C", "OptimizedC", "Scala", "FPCore"),
```

Comment:
Please remove optimizedC from this file. For NN we have to use NNCodegen, not this one.